test/suites/integration/srvexport_port_annotation_targets_test.go

```diff
@@ -87,10 +78,14 @@ var _ = Describe("Port Annotations Targets", func() {
 		}
 
 		testFramework.ExpectDeleted(ctx, service)
-		Eventually(func(g Gomega) {
-			log.Println("Verifying Targets are only craeted for the port defined in Port Annotaion in ServiceExport")
-			targets := testFramework.GetTargets(ctx, targetGroup, deployment)
-			Expect(len(targets) == 0)
-		}).WithTimeout(5*time.Minute + 30*time.Second)
+		Eventually(func() int {
+			log.Println("Eventually deleting targets, getting registered targets")
+			retrievedTargets, err := testFramework.LatticeClient.ListTargetsAsList(ctx, &vpclattice.ListTargetsInput{
+				TargetGroupIdentifier: targetGroup.Id,
+			})
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
We could actually remove the whole:

```
testFramework.ExpectDeleted(ctx, service)
		Eventually(func(g Gomega) {
			log.Println("Verifying Targets are only craeted for the port defined in Port Annotaion in ServiceExport")
			targets := testFramework.GetTargets(ctx, targetGroup, deployment)
			Expect(len(targets) == 0)
		}).WithTimeout(5*time.Minute + 30*time.Second)
```

This test case intend to test "Verifying Targets are only created for the port defined in Port Annotation in ServiceExport",  and the delete k8sService and verify 0 targets don't need to be part of this test

Reply:
Got it, I have removed it now